clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -98,6 +98,11 @@ Improvements to clang-tidy
 New checks
 ^^^^^^^^^^
 
+- New :doc:`bugprone-public-enable-shared-from-this
+  <clang-tidy/checks/bugprone/public-enable-shared-from-this>` check.
+
+  Checks if a class deriving from enable_shared_from_this has public access specifiers, because otherwise the program will crash when shared_from_this is called.
```

Comment:
again, wrong. "Check if class publicly derive from std::enable_shared_from_this ..."
And wrap on 80 column.

Reply:
Right, will run git clang format, thanks for the reminder and feedback

Reply:
ok manually reformatted since git clang format doesn't work on rst, and changed documentation